### DISCUSSION TITLE
Feature request: monitor queue events

### DISCUSSION THREAD

**Discussion by elenakondrateva** [spatie/laravel-schedule-monitor#53] [Ideas]
Currently scheduled jobs monitoring is not very useful if jobs are dispatched on a queue - basically monitor tracks only dispatching itself but not job execution/fail.

```
        $schedule->job(\App\Jobs\Synthetic\ArchiveTestsJob::class)
            ->monthlyOn(1, '5:00')
            ->storeOutputInDb()
        ;
```

Is there any plan to log queue events about monitored jobs as well?

**Comment by freekmurze** [comment]
Currently not, feel free to create a PR

**Reply by elenakondrateva** [reply]
Thanks!
